src/PowerShellEditorServices/Services/Workspace/Handlers/ConfigurationHandler.cs

```diff
@@ -133,6 +133,7 @@ public override async Task<Unit> Handle(DidChangeConfigurationParams request, Ca
             if (!_extensionServiceInitialized)
             {
                 await _extensionService.InitializeAsync().ConfigureAwait(false);
+                _extensionServiceInitialized = true;
```

Comment:
Is this otherwise never set?

Reply:
Nope, not referenced anywhere

Reply:
Wowzers. Wonder why it exists? Do we need it?